### ISSUE TITLE
[Accessibility][Text] Markup uses non-semantic elements to provide text emphasis 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1676]
**Steps to Reproduce**
1. Inspect markup for bold and italicized text in text sample.

**Current Behavior**
In the Type Formatting section, bold text is used in the sentence "Bold can be used to emphasize a word or phrase", but this text is marked up with the `<b>` element instead of the `<strong>` element. Similarly, italic text is marked up with the `<i>` element rather than the `<em>` element. The `<b>` and `<i>` elements only provide visual emphasis, rather than providing semantic emphasis, so the emphasis cannot be conveyed to assistive technology.

**Expected behavior/code**
Text intended to convey strong importance, seriousness, or urgency should be marked up with `<strong>`; text intended to convey stress emphasis should be marked up with `<i>`.

Although the default configuration of NVDA does not convey semantic emphasis using `<strong>` and `<em>` elements, they should be used where appropriate instead of non-semantic `<b>` and `<i>` elements in order to support custom settings and future usage of this semantic information.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.3.1 Info and Relationships

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
This issue is not specific to the Core Components, but to the RTE in general and won't be fixed by the Core Components.
